Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,6 +151,34 @@ private Furniture(Furniture other)
 
     public event Action<IPowerRelated> PowerValueChanged;
 
+    // PathfindingModifier is added to PathfindingCost to get the final amount
```

Comment:
While I was aware of documentation comments, I was not aware they would show up in code completion, that is incredibly useful, especially in a project where people may end up having to use functions, properties, etc... that they're not familiar with.